### PR TITLE
Replace Pi added

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

### Description

Replace Pi using recursion added in cp(competitive programming) folder added.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2772

### CODE REVIEW

File: C-Plus-Plus/cp/replace_pi.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        replacepi(a, i + 4);
+    }
+    //Go to the next position
+    else
+    {
+        replacepi(a, i + 1);
+    }
+}
+
+int main()
```
Comment (bhav09) [inline C-Plus-Plus/cp/replace_pi.cpp:43]: Great work! Can you please comment the code and write down the complexity at the end.

**Comment by gayatri517** [comment]
@bhav09 I have added the complexity and the comments in the code. Please check them.
Thank you.

File: C-Plus-Plus/cp/replace_pi.cpp [hunk 2]
```diff
@@ -0,0 +1,63 @@
+/* Problem statement: 
+You are given a string, you have to replace all the occurrences of "pi" int the string into 3.14.
+Example :
```
Comment (bhav09) [inline C-Plus-Plus/cp/replace_pi.cpp]: Great work! A very small change needs to be made. I/O should be at last, after the code.

**Review by bhav09 — CHANGES_REQUESTED** [review]
Reviewed.

**Comment by gayatri517** [comment]
@bhav09 I have made the necessary changes. Kindly check them.
Thank you.

**Comment by bhav09** [comment]
Great work!

**Comment by HarshCasper** [comment]
@all-contributors bot please add @gayatri517 for code

**Pull request merged**